eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -44,25 +58,52 @@ foreach ($res in $responses)
     LogError "Get-GitHubPullRequests failed with exception:`n$_"
     exit 1
   }
-
   $openPullRequests = $pullRequests | ? { $_.State -eq "open" }
-  if ($openPullRequests.Count -gt 0)
+
+  try {
```

Comment:
if ($CentralRepoId) 
{
  # check central PR
  # If central PR exist and open, then skip
  $pullRequestNumber = $Matches["PrNumber"]

  if (!$pullRequestNumber) { LogError .... }

  // Check central PR status
  // If not closed then continue
  // if error then log and continue

  // close any open PRs
}